2024-01-20 09:16:50,238 INFO    MainThread:38760 [wandb_setup.py:_flush():76] Current SDK version is 0.16.2
2024-01-20 09:16:50,239 INFO    MainThread:38760 [wandb_setup.py:_flush():76] Configure stats pid to 38760
2024-01-20 09:16:50,240 INFO    MainThread:38760 [wandb_setup.py:_flush():76] Loading settings from /root/.config/wandb/settings
2024-01-20 09:16:50,242 INFO    MainThread:38760 [wandb_setup.py:_flush():76] Loading settings from /com.docker.devenvironments.code/kaggle-harmful-brain-activity/data_loader/wandb/settings
2024-01-20 09:16:50,243 INFO    MainThread:38760 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-01-20 09:16:50,244 INFO    MainThread:38760 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-01-20 09:16:50,245 INFO    MainThread:38760 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-01-20 09:16:50,248 INFO    MainThread:38760 [wandb_init.py:_log_setup():526] Logging user logs to /com.docker.devenvironments.code/kaggle-harmful-brain-activity/data_loader/wandb/run-20240120_091650-bffte1f8/logs/debug.log
2024-01-20 09:16:50,249 INFO    MainThread:38760 [wandb_init.py:_log_setup():527] Logging internal logs to /com.docker.devenvironments.code/kaggle-harmful-brain-activity/data_loader/wandb/run-20240120_091650-bffte1f8/logs/debug-internal.log
2024-01-20 09:16:50,250 INFO    MainThread:38760 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f5db4a6b340>
2024-01-20 09:16:50,251 INFO    MainThread:38760 [wandb_init.py:init():566] calling init triggers
2024-01-20 09:16:50,252 INFO    MainThread:38760 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {'lr': 0.001, 'loss': 'KLDivloss'}
2024-01-20 09:16:50,253 INFO    MainThread:38760 [wandb_init.py:init():616] starting backend
2024-01-20 09:16:50,253 INFO    MainThread:38760 [wandb_init.py:init():620] setting up manager
2024-01-20 09:16:50,262 INFO    MainThread:38760 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2024-01-20 09:16:50,265 INFO    MainThread:38760 [wandb_init.py:init():628] backend started and connected
2024-01-20 09:16:50,300 INFO    MainThread:38760 [wandb_run.py:_label_probe_notebook():1294] probe notebook
2024-01-20 09:16:50,304 INFO    MainThread:38760 [wandb_run.py:_label_probe_notebook():1304] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-01-20 09:16:50,306 INFO    MainThread:38760 [wandb_init.py:init():720] updated telemetry
2024-01-20 09:16:50,716 INFO    MainThread:38760 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-01-20 09:16:52,381 INFO    MainThread:38760 [wandb_run.py:_on_init():2254] communicating current version
2024-01-20 09:16:53,849 INFO    MainThread:38760 [wandb_run.py:_on_init():2263] got version response 
2024-01-20 09:16:53,850 INFO    MainThread:38760 [wandb_init.py:init():804] starting run threads in backend
2024-01-20 09:16:54,372 INFO    MainThread:38760 [wandb_run.py:_console_start():2233] atexit reg
2024-01-20 09:16:54,373 INFO    MainThread:38760 [wandb_run.py:_redirect():2088] redirect: wrap_raw
2024-01-20 09:16:54,375 INFO    MainThread:38760 [wandb_run.py:_redirect():2153] Wrapping output streams.
2024-01-20 09:16:54,376 INFO    MainThread:38760 [wandb_run.py:_redirect():2178] Redirects installed.
2024-01-20 09:16:54,379 INFO    MainThread:38760 [wandb_init.py:init():847] run started, returning control to user process
2024-01-20 09:16:54,388 INFO    MainThread:38760 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-20 09:16:54,389 INFO    MainThread:38760 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-20 09:16:54,481 INFO    MainThread:38760 [wandb_init.py:_resume_backend():442] resuming backend
2024-01-20 09:16:54,484 INFO    MainThread:38760 [wandb_watch.py:watch():51] Watching
2024-01-20 09:16:54,502 INFO    MainThread:38760 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-01-20 09:16:54,503 INFO    MainThread:38760 [wandb_init.py:_pause_backend():437] pausing backend
2024-01-20 09:16:54,673 INFO    MainThread:38760 [wandb_init.py:_resume_backend():442] resuming backend
